src/modules/Host.rb

```diff
@@ -358,12 +358,23 @@ def set_names(address, names)
   def load_hosts
     return false if SCR.Read(path(".target.size"), CFA::Hosts::PATH) <= 0
 
-    @hosts = CFA::Hosts.new
-    @hosts.load
+    begin
+      @hosts = CFA::Hosts.new
+      @hosts.load
+
+      # save hosts to check for changes later
+      @hosts_init = CFA::Hosts.new
+      @hosts_init.load
+    rescue RuntimeError => error
```

Comment:
this won't help with IOErrors like file missing. I think it makes sense to add it also there.

Reply:
This https://github.com/yast/yast-network/pull/621/files#diff-4be40ec54e4c9b91ae249f9931c14eb8R359 should help for missing files

Reply:
However adding IOException is definitely good idea

Reply:
it not just about missing files, but also other temporary IO problems that can happen. and it is really `IOError` not IOException